modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -5,34 +5,26 @@ module DecisionReviews
     module NoticeOfDisagreements
       class EvidenceSubmissionsController < AppealsApi::ApplicationController
         skip_before_action :authenticate
-        before_action :set_submission_attributes, if: -> { params[:nod_id] }
+        before_action :set_submission_attributes, only: :create
 
         def create
-          submission = AppealsApi::EvidenceSubmission.create(@submission_attributes)
-          signed_url = submission.get_location
-
-          render json: { data:
-                          {
-                            attributes:
-                              {
-                                status: submission.status,
-                                id: submission.id,
-                                appeal_id: submission.supportable_id,
-                                appeal_type: submission.supportable_type
-                              },
-                            location: signed_url
-                          } }
+          upload = VBADocuments::UploadSubmission.create consumer_name: 'appeals_api_nod_evidence_submission'
```

Comment:
I generally prefer the 'bang' version when creating records at the controller level, as it blows up early if something goes wrong. What do you think?

`create!`